libraries/stdlib/src/kotlin/collections/Collections.kt

```diff
@@ -235,6 +235,30 @@ public fun <T> List<T>.binarySearch(fromIndex: Int = 0, toIndex: Int = size, com
 }
 
 /**
+ * Returns the elements from the list from indexes specified in range.
+ */
+public operator fun <T> List<T>.get(indexesRange: IntRange): List<T> =
+        if (indexesRange.isEmpty())
+            listOf<T>()
+        else if (indexesRange.first < 0)
+            throw IndexOutOfBoundsException("Range beginning (${indexesRange.first}) is less than zero.")
+        else if (indexesRange.last > lastIndex)
+            throw IndexOutOfBoundsException("Range end (${indexesRange.last}) is bigger then the end of table ($lastIndex)")
+        else if (indexesRange.first <= indexesRange.last)
+            subList(indexesRange.first, indexesRange.last + 1).toList()
+        else
+            throw Exception("Unknown exception in getting range of indexes from list")
+
+/**
+ * Returns the list multiplication.
+ */
+public operator fun <T> List<T>.times(factor: Int): List<T> {
+    require(factor >= 0) { "List factor must be bigger then 0" }
```

Comment:
Also, in fact the message is not exact correct: factor shouldn't be negative so zero is also included